package.json

```diff
@@ -23,7 +24,8 @@
     "react-hook-form": "^7.54.1",
     "react-pro-sidebar": "^1.1.0",
     "react-router-dom": "^7.0.2",
-    "react-toastify": "^10.0.6"
+    "react-toastify": "^10.0.6",
+    "styled-components": "^6.1.13"
```

Comment:
Both emotion and styled-components are used for the same purpose, we only need one and the one reccomended from MUI is emotion as it has support for react server comopnents. We can remove the other package.

Reply:
done